apps/settings/style/downloads.css

```diff
@@ -235,6 +235,11 @@
 }
 
 /* RTL rules */
+html:-moz-dir(rtl) #downloadList aside.download-status {
+  right: 0.2rem;
```

Comment:
As the icon is smaller than 30x30, it is aligned to the right. I would suggest to add `background-center` as you did in the previous patch. 
And `#downloadList aside` set `margin-right` to 0.5rem, so the value of right should be -0.3rem in rtl layout.